Userland/Libraries/LibC/signal.cpp

```diff
@@ -75,6 +75,12 @@ int sigaddset(sigset_t* set, int sig)
     return 0;
 }
 
+int sigaltstack(const stack_t* ss, stack_t* old_ss)
```

Comment:
Would be nice to at least add a test case, since there are no users.

Reply:
Great idea, this found a slew of bugs in the kernel + show a major feature was missing from the sigaltstack implementation as well, re-review welcome :)